sympy/polys/matrices/sdm.py

```diff
@@ -1096,6 +1096,48 @@ def lu_solve(A, b):
         """
         return A.from_ddm(A.to_ddm().lu_solve(b.to_ddm()))
 
+    def fflu(self):
+        """
+        Fraction-free LU decomposition of SDM.
+
+        Explanation
+        ===========
+
+        This method computes the PLDU decomposition
+        using Gauss-Bareiss elimination in a fraction-free manner,
+        it ensures that all intermediate results remain in
+        the domain of the input matrix. Unlike standard
+        LU decomposition, which introduces division, this approach
+        avoids fractions, making it particularly suitable
+        for exact arithmetic over integers or polynomials.
```

Comment:
This docstring is duplicating information that is elsewhere. It can just have a reference to the DomainMatrix.fflu method. The only thing worth mentioning here is that this currently just uses the DDM implementation.